src/components/home/ClusterDashboardItem.js

```diff
@@ -22,6 +23,56 @@ import moment from 'moment';
 import PropTypes from 'prop-types';
 import React from 'react';
 import RefreshableLabel from 'UI/refreshable_label';
+import styled from '@emotion/styled';
+
+const WrapperStyles = css`
+  display: flex;
+  background-color: ${props => props.theme.colors.darkBlueLighter1};
+  border-radius: 5px;
+  border: 0px;
+  min-height: 20px;
+  padding: 19px;
+  margin-bottom: 20px;
+  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.05);
```

Comment:
I would say that, reading the error, whihc is quite explicit, this is what is causing the warning. 
AFAIK we use this without the function:

```javascript
background-color: ${props.theme.colors.darkBlueLighter1};
````

the function is for `styled`